evm/bridge-token-factory/test/BridgeToken.js

```diff
@@ -36,10 +36,12 @@ describe('BridgeToken', () => {
     const bridgeToken = await BridgeTokenInstance.deploy()
     await bridgeToken.waitForDeployment()
 
+    const nearBridgeDeriveAddress = "0xa966f32b64caaee9211d674e698cb72100b5e792";
```

Comment:
I'm not sure there is a library. There is a file in examples repo that does the calculations https://github.com/near-examples/near-multichain/blob/main/src/services/kdf.js

Anyway, I think it's not necessary. We hardcode signatures signed by this address, so we can also hardcode the address itself. It should have no affect on the integrity of the tests.

Reply:
Even if it changes we don't need to change the tests. Or if we change the tests we will still have to hardcode signatures. But I guess it can be useful to include for the deploy scripts

Reply:
30524e8d7a9bbe3429981fc21cf13e40a55ca7e6

Reply:
Thanks